drivers/platform/altera/spi.c

```diff
@@ -61,15 +62,29 @@ int32_t spi_init(struct spi_desc **desc,
 		 const struct spi_init_param *param)
 {
 	spi_desc *descriptor;
+	altera_spi_desc *altera_descriptor;
```

Comment:
i know there was a discussion of shifting to intel instead of altera.

Reply:
I would do the renaming in another patch. There are several other drivers that require this update.
See for example: https://github.com/analogdevicesinc/no-OS/tree/master/drivers/axi_core/jesd204